test/FunctionalTests/Tests/DataServices/UnitTests/Client.TDD.Tests/Tests/HttpWebRequestUnitTests.cs

```diff
@@ -31,41 +33,22 @@ public void IsRunningOnSilverlightShouldReturnTrueWhenRunningOnSilverlightOnly()
         [Fact]
         public void SetUserAgentShouldSucceed()
         {
-            HttpWebRequest request = (HttpWebRequest)HttpWebRequest.Create("http://www.svc");
+            HttpRequestMessage request = new HttpRequestMessage(new HttpMethod("GET"), "http://www.svc");

```

Comment:
Maybe add a method create that takes a string and defaults to Get internally.

Reply:
@marabooy  Create is a HttpWebRequest class